sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/Models/ImportJob.Serialization.cs

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.DigitalTwins.Core
+{
+    public partial class ImportJob : IUtf8JsonSerializable
+    {
+        // This class definition overrides deserialization implementation in order to allow finishedDateTime to be null in the response.
```

Comment:
Can you please clarify?

Why doesn't justting the member in the C# object to DateTimeOffset? not allow the response to contain a null value?

Reply:
@sjiherzig  and @abhinav-ghai can help elaborate here, since we talked extensively on this change - but essentially this was a bug we found where we are returning null in the api response but in the swagger this is defined as a non-null property. So the auto-generated code has a check (which I am replacing) that this property should not be null when it will be.

As it turns out, we should never give a null property to customers in an api response - this is what we were told by someone in the api review board. But that will require a change in the bulk code itself. So instead of marking this value as nullable in the swagger, which will require a swagger review and is ultimately not the behavior we even want, we are making this change here to buy us time. Then we will fix the real issue which is that this should never be returned as null, and then this will be removed from the SDK.